### DISCUSSION TITLE
Selectable Workplanes Via Their Face

### DISCUSSION THREAD

**Discussion by TheMakerTales** [hlorus/CAD_Sketcher#64] [Feature Requests]
Currently, you can only select the XYZ workplanes and other workplanes via their edges. It would be great if they were selectable via their faces too.

**Comment by hlorus** [comment]
Agree this would be nice as long as it's still easily possible to select other entities through that face.

Even other workplanes should be selectable if occluded by a workplane face, probably by picking it's edge.
